### ISSUE TITLE
Add build hashes to published release

### ISSUE THREAD

**Issue by stevelacy** [maxgoedjen/secretive#401]
Ideally github actions would retain the SHAs for the continued future, but as it deletes the logs based on repo retention settings we don't have the ability to access the SHAs of the builds.
![Screen Shot 2022-08-13 at 4 26 05 PM](https://user-images.githubusercontent.com/5648999/184516536-24a54516-94e4-46c6-8f11-9a2192e3f746.png)
I would suggest that the github action adds the release hashes to the published release when it is uploaded.

**Comment by sandstrom** [comment]
I would close this issue, both hashes exist in releases, and attestation.

cc @maxgoedjen

**Comment by maxgoedjen** [comment]
Yep as @sandstrom points out this should be much better addressed by release attestation now. :)
